src/librustc_middle/mir/visit.rs

```diff
@@ -1133,6 +1150,8 @@ pub enum NonUseContext {
     StorageDead,
     /// User type annotation assertions for NLL.
     AscribeUserTy,
+    /// Coverage code region and counter metadata.
+    Coverage,
```

Comment:
@richkadel is `PlaceContext::NonUse(NonUseContext::Coverage)` reserved for a future use? It's never constructed currently.

Reply:
@tmiasko I think this got left in accidentally from a prior version of these changes where it was used. I believe it can be removed.